contrib/objcwrapper/src/objcwrapper.nit

```diff
@@ -65,3 +81,8 @@ end
 
 var g = new CodeGenerator(v.model)
 g.generate
+
+if failed_parsing.not_empty then
+	print_error "Failed to parse {failed_parsing.length}/{opts.rest.length} files:"
+	print_error failed_parsing.join(", ")
```

Comment:
I'm not fan of comma-separated filenames as they mess with selection in console. Separate with spaces or newlines (I prefer newlines)